blogg/posts/posts.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
```

Comment:
We don't need this work in the scaffolding piece

Reply:
Ie, we don't need it "to" work.  Have blanked out the code